indico/modules/events/models/events.py

```diff
@@ -555,6 +563,18 @@ def short_external_url(self):
         id_ = self.url_shortcut or self.id
         return url_for('events.shorturl', confId=id_, _external=True)
 
+    @property
+    def map_url(self):
+        if self.own_map_url:
+            return self.own_map_url
+        elif not self.room:
+            return ''
+        return self.room.map_url or ''
+
+    @map_url.setter
+    def map_url(self, value):
+        self.own_map_url = value
```

Comment:
Do we need this setter in the end?

Reply:
Nope, if I also change it the form.

Reply:
yeah, i think `map_url` should only be used in the display areas, and anything else would use the `own_map_url`

as a bonus, you could set the room-provided map url as a placeholder in the custom map url field to make it clear which url is used when you don't enter anything there. it would only be applied after saving in case you edit the room but i don't think this would be an issue

Reply:
Done!